pymc3/step_methods/arraystep.py

```diff
@@ -6,7 +6,13 @@
 from numpy.random import uniform
 from enum import IntEnum, unique
 
-__all__ = ['ArrayStep', 'ArrayStepShared', 'metrop_select', 'Competence']
+__all__ = [
+    'ArrayStep', 'ArrayStepShared', 'metrop_select',
+    'Competence', 'SamplingError']
+
+
+class SamplingError(RuntimeError):
```

Comment:
👍  we could probably do with either `pymc3.exceptions` or `pymc3.sampling.exceptions` soon